### DISCUSSION TITLE
using include tag.html and where site.tags actually lives

### DISCUSSION THREAD

**Discussion by bvoightlab** [greenelab/lab-website-template#231] [General] [answered]
Hi all - In previous version of the template (like 0.6.2), one could use pubtags.html to create (and list) a set of tags that users could click and sort content for a given page. i.e. a default list that the creator could specify.

But it looks like that structure is gone in the 1.6 version, which seems fine since it looks like there's a {% include tags.html} functionality that could do the job

However, I can not for the life of me get that to work -- i.e., see

https://voightlab.com/tools/

code is:
https://github.com/bvoightlab/bvoightlab.github.io/blob/main/tools/index.md

### 
In a separate note, if you look at this page:

https://voightlab.com/research/

code is: 
https://github.com/bvoightlab/bvoightlab.github.io/blob/main/research/index.md

here, the call to tag.html refers to "tags=site.tags" which clearly derives from the default template

but I can not for the life of me (after grepping) find the place where site.tags can be edited or where those are included from

I'd be fine if there were multiple bucket like "site1.tags" , "site2,tags" to which I could edit and create/refer groups of tags
or alternatively, use {% include tags.html} with a listing of tags as the documentation seems to suggest that I could do

I'm almost certainly missing something obvious, so please point me to higher ground!

Thanks,
-BV

**Comment by vincerubinetti** [comment, marked as answer]
> one could use pubtags.html 

I'm confused by this, there was never a component called that, but there was and still is a `tags.html` component, and [you can use it to specify a specific list of tags](https://greene-lab.gitbook.io/lab-website-template-docs/basics/components/tags), but you already appear to be doing that correctly on your tools page and it's not working.

I've looked into this and it is unfortunately a bug. A quick fix you can do for now is changing `return []` to `return object` in `/_plugins/misc.rb`. (Untested, but I'm pretty sure that will work). I will create an issue for this.

As for `site.tags`, [that's a Jekyll thing](https://jekyllrb.com/docs/posts/#tags). Remember that this template is based on Jekyll, and much of its functionality comes from there.

**Reply by vincerubinetti** [reply]
https://github.com/greenelab/lab-website-template/issues/232
